scout/server/blueprints/variants/controllers.py

```diff
@@ -48,6 +48,14 @@ def variants(store, institute_obj, case_obj, variants_query, page=1, per_page=50
     for variant_obj in variant_res:
         overlapping_svs = [sv for sv in store.overlapping(variant_obj)]
         variant_obj['overlapping'] = overlapping_svs or None
+
+        # Get all previous ACMG evalautions of the variant
+        evaluations = []
+        for evaluation_obj in store.get_evaluations(variant_obj):
```

Comment:
If it is slow we could add an index, lets use it for a while and see

Reply:
I don't think it should slow, but that *would* be a very straightforward index otherwise! Wonderful incentive to classify variants! Let's just document / legend this well so that we don't confuse anyone.

Reply:
Waiting for bootstrap 4 sounds like a good idea, as it would allow for better styling of the badges. Any ideas on how to style it to avoid misunderstandings? The tooltip and different color is maybe not enough?